### DISCUSSION TITLE
Thanks Team

### DISCUSSION THREAD

**Discussion by mxmissile** [linq2db/linq2db#4104] [General]
I just have to say thanks to the linq2db team, I am mid way through a Blazor Server project and started with linq2db from the start, this has been absolutely the best DB access experience have seen. Zero ceremony, zero overhead, extremely fast, and 99% of the time things just work the first time.

Thanks Team!

**Comment by viceroypenguin** [comment]
Thanks @mxmissile! So glad you are enjoying it.

**Comment by igor-tkachev** [comment]
@mxmissile, thanks. The secret is simple - we develop it for ourselves and become the very first users.
